yucca/training/configuration/configure_callbacks.py

```diff
@@ -110,9 +113,10 @@ def get_loggers(
                 notes=run_description,
                 save_dir=version_dir,
                 project=project,
-                group=task,
+                group=experiment if experiment != "default" else task,
```

Comment:
I think we should stick with 1 grouping, and I do not mind always grouping by experiment (even if it is default), since you can always make custom grouping in WandB, so this is mostly a QOL improvement for those that wish to use specific groups, which I suspect might be those that will also be using the "experiment" variable a lot

Reply:
Roger! Thanks!